src/main/java/org/elasticsearch/index/translog/TranslogStats.java

```diff
@@ -70,10 +80,16 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
     @Override
     public void readFrom(StreamInput in) throws IOException {
         estimatedNumberOfOperations = in.readVInt();
+        if (in.getVersion().onOrAfter(Version.V_1_5_0)) {
+            translogSizeInBytes = in.readLong();
+        }
     }
 
     @Override
     public void writeTo(StreamOutput out) throws IOException {
         out.writeVInt(estimatedNumberOfOperations);
+        if (out.getVersion().onOrAfter(Version.V_1_5_0)) {
+            out.writeLong(translogSizeInBytes);
```

Comment:
My rational is that this is not a high performance API, so I tend not to use writeVLong because of it's limitation regarding negative numbers. Will change though, as here it's possible.